### DISCUSSION TITLE
[Projects Beta] More granular workflows for `Item added to project`

### DISCUSSION THREAD

**Discussion by marina-chibizova** [community/community#49737] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Currently one can only set one workflow for adding new items to a project. But issues and pull requests have different status upon creation - issues will naturally fit into `To do` or `Triage` status, while pull requests are `Ready for review`.  So is it possible to make this workflow more granular? Or just allow duplicating it.

![Screen recording 2023-03-10 12 59 01 PM](https://user-images.githubusercontent.com/80579198/224404046-583fe86f-416a-4516-8de8-90211d29aa0d.gif)

**Comment by Bexolo** [comment]
We would agree with this. 

We would like way more control on what happens to issues when they're added to a project. Our particular use case includes needing to set a custom 'Triage' status when a bug is raised, so that it is automatically filtered into a seperate Bug Triage view for the QA team.

We would also like it when an issue added to project, assign a selected label.
